src/census.app.src

```diff
@@ -0,0 +1,16 @@
+{application, census,
+ [{description, "Erlang stats collection and distributed tracing framework"},
+  {vsn, "0.1.0"},
+  {registered, []},
+  {mod, {census_app, []}},
```

Comment:
we use opencensus as the name of the project/modules usually.

Reply:
Ok, I'll rename the app and modules to use `opencensus` instead of `census`.